### ISSUE TITLE
Improve localization (op: Change date language)

### ISSUE THREAD

**Issue by melvinsalas** [theNewDynamic/gohugo-theme-ananke#296]
Hi,

I configure a project with spanish as main language

```Javascript
title = "..."
baseURL = "https://..."
languageCode = "es"
theme = "ananke"
```

but the date is showed in english

![](https://user-images.githubusercontent.com/708747/84707349-c5755400-af1b-11ea-82d9-6b055d72426a.png)

How can I change it?

**Comment by budparr** [comment]
We have plans to do more localization of the theme. I'll leave this open.

**Comment by peter-jerry-ye** [comment]
@melvinsalas I think you need to put "DefaultContentLanguage = "es"" in your configuration, as the example site did

**Comment by peter-jerry-ye** [comment]
@budparr in your example site you are using "DefaultContentLanguage ="en"", does that means you are not using country code?

**Comment by peter-jerry-ye** [comment]
By the way, I think this problem should be fixed by #291
